### ISSUE TITLE
Auto-completes non-re-exported modules from other crates

### ISSUE THREAD

**Issue by renatoathaydes** [intellij-rust/intellij-rust#1877]
I have a crate that depends on `difference`.

In the integration tests for this crate, when I type `use my_crate::` the IDE suggests `difference` even though `difference` is not re-exported.

Using that won't compile. The IDE should not suggest that.

The fact that using that does not cause an error to be shown in the IDE is probably a separate bug?!

In my crate, I do this at the root-level:

```rust
extern crate difference;
```

I would expect `difference` to be visible from my crate only if I had done:

```rust
pub extern crate difference;
```

**Comment by Undin** [comment]
It's not reproduced anymore
